docs/patterns/flashing.rst

```diff
@@ -110,7 +110,7 @@ categories.  The loop looks slightly different in that situation then:
 
 .. sourcecode:: html+jinja
 
-   {% with messages = get_flashed_messages(with_categories=true) %}
+   {% with messages = get_flashed_messages(with_categories=True) %}
```

Comment:
lowercase versions of true/false/none are the default in jinja, the python-cased variants are just there as an alias.

Reply:
thanks for this, I didn't realise! True had always worked for me and I never realised lowercase was the default